### PR TITLE
feat!: Add view of all system triggers on Scheduled Jobs page

### PR DESCRIPTION
**Pull request by drbgfc**
[#OCD-3159]

### CODE REVIEW

File: src/app/pages/administration/administration.state.js [hunk 1]
```diff
@@ -195,6 +195,13 @@ let states = {
                     }
                     return [];
                 },
+                scheduledSystemJobs: (authService, networkService) => {
+                    'ngInject'
+                    if (authService.hasAnyRole(['ROLE_ADMIN', 'ROLE_ONC', 'ROLE_ACB'])) {
```
Comment (andlar) [inline src/app/pages/administration/administration.state.js]: ROLE_ACB can't see any of these jobs, so they shouldn't be in this If

File: src/app/services/network.service.js [hunk 2]
```diff
@@ -393,6 +393,10 @@ export class NetworkService {
         return this.apiGET('/schedules/triggers');
     }
 
+    getScheduledSystemJobs () {
```
Comment (andlar) [inline src/app/services/network.service.js]: Should add a test to the network.service.spec.js file

File: src/app/services/network.service.js [hunk 3]
```diff
@@ -393,6 +393,10 @@ export class NetworkService {
...
+        return this.apiGET('/schedules/triggers/?jobType=system');
```
Comment (andlar) [inline src/app/services/network.service.js]: I don't think the last `/` should be there. The `/`s imply a sub-level of stuff, while the `?` query is a filter

File: src/app/pages/administration/administration.state.js (hunk 1 above)
Comment (tmy1313) [inline src/app/pages/administration/administration.state.js]: Just want to confirm that an ACB can see this data?

File: src/app/services/network.service.js (hunk 3 above)
Comment (drbgfc) [reply src/app/services/network.service.js]: Indeed - fat-fingered but surprisingly it works fine - will remove

File: src/app/pages/administration/administration.state.js (hunk 1 above)
Comment (drbgfc) [reply src/app/pages/administration/administration.state.js]: removed, thanks

Comment (drbgfc) [reply src/app/pages/administration/administration.state.js]: No, they can't, thanks. I considered in front-end but copy-pasting caused this gloss-over here

File: src/app/services/network.service.js (hunk 2 above)
Comment (drbgfc) [reply src/app/services/network.service.js]: added

File: src/app/pages/administration/jobs/scheduled/triggers.html [hunk 4]
```diff
@@ -24,3 +24,27 @@ <h3>Currently Scheduled Jobs</h3>
...
+        <th scope="col">Description</th>
+        <th scope="col" class="search-header" st-sort="nextRunDate" st-skip-natural="true" st-sort-default="true">Next Run Date</th>
+        <th scope="col" class="search-header" st-sort="triggerScheduleType" st-skip-natural="true">Trigger Schedule Type</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr ng-repeat="scheduledSystemJob in $ctrl.displayedScheduledSystemJobs track by $index">
+        <td>{{ scheduledSystemJob.name }}</td>
+        <td>{{ scheduledSystemJob.description }}</td>
+        <td>{{ scheduledSystemJob.nextRunDate | date:'yyyy-MM-dd HH:mm:ss Z' }}</td>
```
Comment (andlar) [inline src/app/pages/administration/jobs/scheduled/triggers.html]:
```suggestion
        <td>{{ scheduledSystemJob.nextRunDate  ? (scheduledSystemJob.nextRunDate | date:'yyyy-MM-dd HH:mm:ss Z') : 'In Progress' }}</td>
```

**Pull request merged**